### ISSUE TITLE
[BUG]Possible Bug - Too High Logistics Carrier Engine Tech May Cause Bugs

### ISSUE THREAD

**Issue by AnakinShieh** [Windows10CE/DSPPlugins#4]
**Which of my mods you were using**
DSPCheats

**Describe the bug**
Unlock technology would default unlock this technology Logistics Carrier Engine to 10000, but that will cause the game unplayable mechanically. Since the higher this technology is, the more energy it costs for interstellar logistics. When reaching at some level, the default maximum energy stored wouldn't be sufficient for a single trip.

Also, it seems that even without this issue, for example, when I enable the no use for power option provided in this mod, the interstellar ship could take off, but it seems that due to the too high speed, it doesn't work as normally. For example, it would move quickly around the sun like electrons but never landed and take the stuffs back.

I don't think the bug is related to this mod, but to make the game playable when using this cheat mod, I think it's better to set a limit for this technology/upgrades when using this mod to unlock all, and doesn't apply research immediately to infinite for this technology/upgrade.

**Comment by xalunda** [comment]
`move quickly around the sun like electrons but never land and take the stuffs back`
I can also confirm this.
But then again, might not be a mod bug as DSP developers might not have yet implement the 10000% speed it does at this level.

I would too recommend a limit to these endgame upgrades.

**Comment by rbrtgrmn** [comment]
I have to agree with the above comments. the interstellar vessels don't work at the high levels; mine just don't launch. I have spent a pile of time so please add a fix to the code to allow for repair of save file.
